api/api.proto

```diff
@@ -3229,6 +3238,73 @@ message SdkNodeInspectRequest {
   string node_id = 1;
 }
 
+// Options for draining volume attachment from a node
+message SdkNodeDrainAttachmentOptions {
+    // Selector is used for selecting volumes whose
+    // attachment needs to be moved from this node.
+    // The selector could be a list of volume label
+    // key value pairs to select a subset of volumes.
+    map<string, string> selector = 1;
```

Comment:
for all selectors, use `LabelSelectorRequirement` (see examples in other places in the proto file).

Recently, we have been using this for all new selector use cases as it's more extensible.

Same for all selectors in the PR

Reply:
done